antergos/vivaldi/PKGBUILD

```diff
@@ -260,6 +260,5 @@ package() {
 	# ALPM Hooks
 	mkdir -p "${pkgdir}/usr/share/libalpm"/{hooks,scripts}
 	cp "${srcdir}"/../*.hook "${pkgdir}/usr/share/libalpm/hooks"
-	install -Dm755 "${srcdir}"/../vivaldi-widevine-helper "${pkgdir}/usr/share/libalpm/scripts/vivaldi-widevine-helper"
+	install -Dm755 "${srcdir}/../vivaldi-widevine-helper" "${pkgdir}/usr/share/libalpm/scripts/vivaldi-widevine-helper"
```

Comment:
This isn't correct syntax. If you put relative path in quotes it will be treated as string literal.

Reply:
That doesn't make sense though..That would mean the command on the live above it succeeded using the same path. I mean if the problem was with the format of the path then that command would have failed first :confused: